Dockerfile

```diff
@@ -19,7 +19,6 @@ EXPOSE 8080
 EXPOSE 5050
 
 ENV GIT_HASH $GIT_HASH
-# TODO: remove this
 ENV HELM_DEBUG 1
```

Comment:
I decided this is useful enough to keep, and only is noisy during failures, which is very helpful to always have on.